src/python/pants/backend/python/providers/python_build_standalone/rules.py

```diff
@@ -110,19 +342,25 @@ class PBSPythonProviderSubsystem(Subsystem):
             f"""
             Known versions to verify downloads against.
 
-            Each element is a pipe-separated string of `version|platform|sha256|length|url`, where:
+            Each element is a pipe-separated string of either `py_version+pbs_release_tag|platform|sha256|length|url` or
+            `url|sha256|length`, where:
```

Comment:
Looking at this docs, what do you think about instead having the structure be:

```suggestion
            `sha256|length|url`, where:
```

so that the 3-field and 5-field versions are identical other than the `py_version+pbs_release_tag|platform|` field _prefix_ (i.e. less fuss to change between them).

Reply:
My thought here was to make the URL more prominent in the config by putting it the first position (with the trade off of it being a departure from the 5 field form). I'm not wedded to that thought though. I see your point about making it easier to transition between the forms.

Reply:
I'll make the change for now, but would love some feedback on whether the developer UX would be better with URL in the more prominent first position.

Reply:
Ah, yeah, I see what you mean. Neither option is perfect 🤔

I'm torn, so I guess either is fine! Since you've made the change, happy to go with it, but also happy if you want to revert it.

Reply:
Let's go with it. I'm going to ask on Slack what people think of the ordering, but makes sense to get this PR landed and then follow up if we decide to change the order.